cmd/neofs-ir/internal/validate/config.go

```diff
@@ -0,0 +1,213 @@
+package validate
+
+import "time"
+
+type validConfig struct {
```

Comment:
What bugs me is that we have this structure used just for validation while in fact we also have similar structures in various subcomponents. Can those be reused here to avoid duplication? I mean we can _always_ parse into this config and then operate with components (just like NeoGo does).